_includes/header.html

```diff
@@ -76,9 +76,9 @@
       {% endfor %}
     </ul>
 
-    {% if site.disable_search != true %}
+    {% if site.search_provider == 'github' %}
     <!-- GitHub Search -->
-    <form class="form-inline mr-md-3" action="{{ site.github_url }}/search">
+    <form class="form-inline mr-md-3" action="https://github.com/openSUSE/{{ site.search_github_repo }}/search">
```

Comment:
The templates might be used for YaST website at some point, which lives in yast org on github

Reply:
Fixed. Now the `search_github_repo` option need to be `openSUSE/news-o-o` or `yast/yast`.